src/internal/ClickAwayListener.js

```diff
@@ -0,0 +1,69 @@
+// @flow weak
+
+import { Component, PropTypes } from 'react';
+import ReactDOM from 'react-dom';
+import events from '../utils/events';
```

Comment:
I'm wondering if we need those helpers. I think that we have alternatives already present.